veracruz-client/src/admin_client.rs

```diff
@@ -0,0 +1,145 @@
+//! The Veracruz admin client
+//!
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Licensing and copyright notice
+//!
+//! See the `LICENSE_MIT.markdown` file in the Veracruz root directory for
+//! information on licensing and copyright.
+
+use crate::error::VeracruzClientError;
+use std::convert::identity;
+use std::num::NonZeroU32;
+use std::time::Duration;
+use veracruz_utils::policy::policy::Policy;
+use serde::{Serialize, Deserialize};
+
+
+/// Provides VeracruzAdminClient, a client API for administrative actions
+/// that don't require a policy
+///
+/// For interaction with the actual enclave see VeracruzClient
+///
+#[derive(Debug)]
+pub struct VeracruzAdminClient {
+    url: String,
```

Comment:
Should this be a `String` or an `Url` type?

Reply:
Do we have a `Url`? I don't think Rust has one. The closest would be `SocketAddr`, but this doesn't preserve the human readable representation.